tests/python_client/milvus_client/test_milvus_client_collection.py

```diff
@@ -1188,27 +1409,726 @@ def test_milvus_client_load_loaded_collection(self):
+        partition_name_1 = cf.gen_unique_str("partition1")
+        partition_name_2 = cf.gen_unique_str("partition2")
+        
+        self.create_collection(client, collection_name, default_dim)
+        self.create_partition(client, collection_name, partition_name_1)
+        self.create_partition(client, collection_name, partition_name_2)
+        
+        self.load_collection(client, collection_name)
+        self.release_partitions(client, collection_name, [partition_name_1])
+        self.drop_partition(client, collection_name, partition_name_1)
+        self.load_collection(client, collection_name)
+        
+        # Query on the remaining partition
+        self.query(client, collection_name, filter=default_search_exp,
+                   partition_names=[partition_name_2])
+        
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L0)
+    def test_milvus_client_load_collection_after_index(self):
+        """
+        target: test load collection, after index created
+        method: insert and create index, load collection with correct params
+        expected: no error raised
+        """
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        # Create collection
+        self.create_collection(client, collection_name, default_dim, consistency_level="Strong")
+        self.release_collection(client, collection_name)
+        self.drop_index(client, collection_name, "vector")
+        # Insert data
+        rng = np.random.default_rng(seed=19530)
+        rows = [{default_primary_key_field_name: i, default_vector_field_name: list(rng.random((1, default_dim))[0]),
+                 default_float_field_name: i * 1.0, default_string_field_name: str(i)} for i in range(default_nb)]
+        self.insert(client, collection_name, rows)
+        # Prepare and create index
+        index_params = self.prepare_index_params(client)[0]
+        index_params.add_index(field_name="vector", index_type="IVF_SQ8", metric_type="L2")
+        self.create_index(client, collection_name, index_params)
+        # Load and release collection
+        self.load_collection(client, collection_name)
+        self.release_collection(client, collection_name)
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_milvus_client_load_empty_collection(self):
+        """
+        target: test load an empty collection with no data inserted
+        method: no entities in collection, load and release the collection
+        expected: load and release successfully
+        """
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        # Create collection
+        self.create_collection(client, collection_name, default_dim, consistency_level="Strong")
+        self.release_collection(client, collection_name)
```

Comment:
i don't think we need this case, because if creating a collection in fast mode, it was already indexed and loaded by default.

Reply:
if you want, we can verify this default behavior.